addons/mrp/views/mrp_bom_views.xml

```diff
@@ -189,8 +189,7 @@
                     <field name="product_id" groups="product.group_product_variant" optional="hide"/>
                     <field name="company_id" groups="base.group_multi_company" optional="show"/>
                     <field name="product_qty" optional="hide"/>
-                    <field name="product_uom_category_id" column_invisible="True"/>
```

Comment:
This field is needed in the domain 
 https://github.com/odoo-dev/odoo/blob/bda9f52d1ac7a36b55ce0ca742cbc9cfa634a06b/addons/mrp/models/mrp_bom.py#L50

Reply:
Hello, @Whenrow I couldn't find any diff that you have mentioned above, let me know if I am missing something!

Reply:
It was in the `product_uom_id` field definition. but I see you removed the diff in this view file :+1: